geo-types/CHANGES.md

```diff
@@ -1,6 +1,6 @@
 # Changes
 
-## unreleased
```

Comment:
this is the wrong changelog change

Reply:
Good catch! I've updated the geo CHANGES file.

(note that this PR still contains updates to the geo-types CHANGES file by virtue of it being based on #989)